src/main/resources/component/defaultResources/postprocess/TCL/content1.xml

```diff
@@ -0,0 +1,66 @@
+<content>
+	<description>Sample - About Us</description>
+	<typeCode>TCL</typeCode>
+	<typeDescription>2 columns</typeDescription>	
+	<mainGroup>free</mainGroup>
+	<attributes>
+    <attribute xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="jaxbResourceAttribute">
+			<type>Image</type>
+			<name>image</name>
+			<resource>
+				<text>Entando at Work</text>
+				<path>/test-image-rest/resources/cms/images/sample-image-1_d0.jpg</path>
+				<resourceId>entandoAtPlan</resourceId>
+				<restResourcePath>http://localhost:8080/test-image-rest/api/rs/en/jacms/image?id=entandoAtPlan</restResourcePath>
+			</resource>	
+		</attribute>
+		<attribute xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="jaxbTextAttribute">
+			<name>title</name>
+			<type>Text</type>
+			<value>A Modern Platform for Modern UX</value>
+			<roles>
+				<role>jacms:title</role>
+			</roles>
+    </attribute>
+    <attribute xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="jaxbHypertextAttribute">
+			<name>subtitle</name>
+			<type>Hypertext</type>
+			<htmlValue>
+			<![CDATA[
+			By integrating leading edge open source projects and extending their capabilities, Entando allows developers and operations teams to create modern UX on Kubernetes.
+			]]>
+			</htmlValue>
+		</attribute>
+		<attribute xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="jaxbHypertextAttribute">
+			<name>col1</name>
+			<type>Hypertext</type>
+			<htmlValue>
+			<![CDATA[
+			<p class="tcol-content-paragraph">
+        Entando is an open source software company providing the leading micro frontend platform building enterprise web apps on Kubernetes. 
+The company,  founded in 2010 as an open-source system integrator, was re-founded as a product company in 2015 in response to the growing demand for tools and services  to create modern online experiences. Since then, Entando has stepped into international markets. It’s headquartered in San Diego, California with R&D and sales offices in Europe, and features teams all over the world, including the United States, Italy, Brazil, South Africa, Ukraine, and the Philippines.
```

Comment:
```suggestion
The company,  founded in 2010 as an open-source system integrator, was re-founded as a product company in 2015 in response to the growing demand for tools and services  to create modern online experiences. Since then, Entando has stepped into international markets. It’s headquartered in San Diego, California with R&D and sales offices in Europe, and features teams all over the world, including the United States, Italy, Brazil, South Africa, Georgia and the Philippines.
```